integreat_cms/cms/models/users/user_fido_key.py

```diff
@@ -5,9 +5,9 @@
 from ..abstract_base_model import AbstractBaseModel
 
 
-class UserMfaKey(AbstractBaseModel):
+class UserFidoKey(AbstractBaseModel):
```

Comment:
For consistency reasons, we could also rename the model to just `FidoKey` (since it's also called `Organization` instead of `UserOrganization`, `Role` instead of `UserRole` etc...)
But not a bit deal, I'm also fine with leaving it as it is.